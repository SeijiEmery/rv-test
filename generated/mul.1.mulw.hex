0000000    b3  02  85  02  33  14  85  02  33  83  95  02  b3  94  95  02
0000020    b3  03  26  03  33  19  26  03  33  8e  36  03  b3  99  36  03
0000040    73  00  10  00                                                
0000044
