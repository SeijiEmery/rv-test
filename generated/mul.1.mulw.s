mul  t0, a0, s0
mulh s0, a0, s0
mul  t1, a1, s1
mulh s1, a1, s1
mul  t2, a2, s2
mulh s2, a2, s2
mul  t3, a3, s3
mulh s3, a3, s3
nop
nop
nop
nop
nop
ebreak
