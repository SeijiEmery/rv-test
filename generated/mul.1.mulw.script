load /x 0 generated/mul.1.mulw.hex
writereg 10 1
writereg 11 -1
writereg 12 35444612188192495
writereg 13 -35444612188192495
writereg 8 -1
writereg 9 -1
writereg 18 35444612188192495
writereg 19 -35444612188192495
run 0 100000
readreg 5
readreg 6
readreg 7
readreg 28