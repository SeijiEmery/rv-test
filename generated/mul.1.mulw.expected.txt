R5 = -1
R6 = 1
R7 = 48272770088149793
R28 = 48272770088149793
